crates/pixi_core/src/workspace/workspace_mut.rs

```diff
@@ -251,6 +251,35 @@ impl WorkspaceMut {
             let pixi_spec =
                 PixiSpec::from_nameless_matchspec(nameless_spec.clone(), &channel_config);
 
+            // Check if the spec is trying to access an unavailable channel
+            if let PixiSpec::DetailedVersion(spec) = &pixi_spec {
+                if let Some(channel) = &spec.channel {
+                    let workspace = self.workspace();
+                    let workspace_channels = &workspace.workspace.value.workspace.channels;
+                    let config = workspace.channel_config();
+
+                    let found = workspace_channels.iter().any(|c| {
+                        c.channel.clone().into_base_url(&config)
+                            == channel.clone().into_base_url(&config)
+                    });
+
+                    if !found {
+                        let help_cmd = format!("pixi workspace channel add {}", channel);
+                        let help_msg = format!(
+                            "If you are trying to add a new channel to this workspace, use:\n\n  {}",
```

Comment:
Not sure if that is always the user's intention, maybe change into: `To add the missing channel to this workspace use:`?